packages/clerk-js/src/ui/elements/ApplicationLogo.tsx

```diff
@@ -49,7 +49,8 @@ export const ApplicationLogo = (props: ApplicationLogoProps) => {
       sx={{
         display: loaded ? 'inline-block' : 'none',
         height: '100%',
-        width: 'auto',
+        width: '100%',
+        objectFit: 'contain',
```

Comment:
This means that if they upload a photo that is a rectangle, there will be awkward negative space where the logo is rendered. Is that expected/desired? Also seeing cover usage in https://github.com/clerk/javascript/commit/63373bf218d7e20932728f8908f90316d34dec07

Reply:
Do you have a visual example of what you mean? I think `cover` makes sense for an avatar